Resources/Prototypes/Recipes/Crafting/Graphs/improvised/makeshiftstunprod.yml

```diff
@@ -13,12 +13,13 @@
               icon:
                 sprite: Objects/Power/power_cells.rsi
                 state: small
-            - tag: Handcuffs
+            - tag: Cablecuffs
```

Comment:
i dont want to be a pedant (she says as she becomes a pedant) but couldn't this be done with `material: Cablecuffs` with an amount of 1 instead of needing a unique tag? if you tried this and it didnt work then thats fine too